lstchain/data/data_selection_cuts.json

```diff
@@ -0,0 +1,22 @@
+{
+  "events_filters":{
+    "intensity": [200, Infinity],
+    "r": [0, 1],
+    "wl": [0.1, 1],
+    "leakage_intensity_width_2": [0, 0.2]
+  },
+  "fixed_cuts":{
+    "gh_score": [0.6,1],
+    "theta_cut": {"value": 0.2, "unit": "deg"},
```

Comment:
Why do you put 0.2? I though the theta cut was 0.14 as we discussed a few months ago? (but maybe I missed a point)

Reply:
Yes, this I had changed for my own analysis, where I loosened the theta cut for the data selected. This is just a fixed user-dependent cut for now. When we include the sensitivity calculations with the scripts, it will be changed accordingly.